pkg/pillar/types/locationconsts.go

```diff
@@ -46,6 +46,11 @@ const (
 	// UUIDFileName - device UUID
 	UUIDFileName = IdentityDirname + "/uuid"
 
+	// APImaybeV2 - user can statically allow to try API v2
+	APImaybeV2 = IdentityDirname + "/api-maybe-v2"
+	// ServerCertName - filename for server leaf certificate
+	ServerCertName = CertificateDirname + "/server.cert0.pem"
```

Comment:
I would call this Filename and not Name.

Reply:
Ok.